src/foundry/client/pixi/core/loader.d.ts

```diff
@@ -1,26 +1,29 @@
+// FOUNDRY_VERSION: 10.291
+
 import type { ConfiguredDocumentClassForName } from "../../../../types/helperTypes";
 
 declare global {
   /**
-   * A Loader class which helps with loading video and image textures
+   * A Loader class which helps with loading video and image textures.
    */
   class TextureLoader {
-    /**
-     * The cached mapping of textures
-     */
-    cache: Map<string, { tex: PIXI.BaseTexture; time: number }>;
-
     /**
      * The duration in milliseconds for which a texture will remain cached
      * @defaultValue `1000 * 60 * 15`
      */
     static CACHE_TTL: number;
 
+    /**
+     * The cached mapping of textures
+     * @internal
+     */
+    protected #cache: Map<string, { tex: PIXI.BaseTexture; time: number }>;
```

Comment:
The protected can be removed here. I think the JS-Private even forbids that.

Reply:
Updated